cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/CartApi.java

```diff
@@ -0,0 +1,38 @@
+package com.cloudesire.platform.apiclient.api;
+
+import com.cloudesire.platform.apiclient.dto.model.dto.CartDTO;
+import com.cloudesire.platform.apiclient.query.ResellerCatalogQuery;
+import retrofit2.Call;
+import retrofit2.http.Body;
+import retrofit2.http.DELETE;
+import retrofit2.http.GET;
+import retrofit2.http.POST;
+import retrofit2.http.PUT;
+import retrofit2.http.Path;
+import retrofit2.http.QueryMap;
+
+import java.util.List;
+
+public interface CartApi
+{
+    @POST( "cart" )
+    Call<CartDTO> create( @Body CartDTO cart );
+
+    @POST( "cart" )
+    Call<CartDTO> create( @Body CartDTO cart, @QueryMap ResellerCatalogQuery query );
```

Comment:
a che serve ResellerCatalogQuery qui?

Reply:
in un reseller marketplace vengono mandati `reseller` o `catalogId` come parametri grazie ai quali il carrello viene collegato a un reseller catalog